031-DevOpsWithGitHub/Coach/Solution-00.md

```diff
@@ -0,0 +1,21 @@
+# Challenge 00 - Setup Azure & Tools - Coach's Guide
```

Comment:
Hi @colinbeales, the Coach's guide didn't have Coach Guide for Challenge 00.  I've stubbed one out here.

Can you please fill this out and document how the scripts in the /Coach/Setup folder should be used.  I see there are parameters a coach would need to fill out.   Also, it's important to call out the necessary Azure/GH permissions a Coach would need to run the script in their Azure sub.   (From skimming the script, I don't think it would be common for an average Azure user to have the permissions needed to interact with Azure AD?)

Thanks,
-Peter

Reply:
Thanks!  This looks great!